cli/validate/validate.go

```diff
@@ -0,0 +1,100 @@
+package validate
+
+import (
+	"context"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli/validate/cai"
+	"github.com/forseti-security/config-validator/pkg/api/validator"
+	"github.com/forseti-security/config-validator/pkg/gcv"
+	"github.com/pkg/errors"
+	"google.golang.org/api/cloudresourcemanager/v1"
+	"path/filepath"
+	"strings"
+)
+
+// ValidateAssets instantiates GCV and audits CAI assets using "policies"
+// and "lib" folder under policyRootPath.
+func ValidateAssets(assets []cai.Asset, policyRootPath string) (*validator.AuditResponse, error) {
```

Comment:
how user will set policyRootPath? in terraform-validator it's POLICY_PATH env var

Reply:
Via additional option